content/pt-br/docs/concepts/workloads/pods/init-containers.md

```diff
@@ -0,0 +1,266 @@
+---
+reviewers:
+- erictune
```

Comment:
The reviewers section can be removed from localized documentation.

Reply:
Hello my name is Daniela and I just fixed this!